src/molecules/form-fields/checkbox-form-field.test.tsx

```diff
@@ -20,4 +20,44 @@ describe("CheckboxFormField", () => {
         // Assert
         expect(getByLabelText(expected)).not.toBeNull();
     });
+
+    test("when errorsMessage prop set, renders with error message", () => {
+        // Arrange
+        const expected = faker.random.words();
```

Comment:
Same comment from your other PR - since we're not actually asserting on this value (or even using it, really), I would probably just name this `label`.